packages/react/src/primitives/shared/utils.ts

```diff
@@ -1,116 +1,21 @@
 import * as React from 'react';
-import autoprefixer from 'autoprefixer';
-import postcssJs from 'postcss-js';
 
 // Note: this makes nanoid more performant, not less secure
 // @see https://www.npmjs.com/package/nanoid#user-content-non-secure
 import { customAlphabet } from 'nanoid/non-secure';
 const nanoid = customAlphabet('1234567890abcdef', 12);
 
-import {
-  ComponentPropsToStylePropsMap,
-  ComponentPropToStyleProp,
-  ViewProps,
-} from '../types/index';
-
-import { getValueAtCurrentBreakpoint } from './responsive/utils';
-import { useBreakpoint } from './responsive/useBreakpoint';
-import { Breakpoint, Breakpoints } from '../types/responsive';
-
-import { useTheming } from '../../theming/';
-
-export const prefixer = postcssJs.sync([autoprefixer]);
-
 export const strHasLength = (str: unknown): str is string =>
   typeof str === 'string' && str.length > 0;
 
-export const usePropStyles = (props: ViewProps, style: any) => {
-  const {
-    theme: {
-      breakpoints: {
-        values: breakpoints,
-        unit: breakpointUnit,
-        defaultBreakpoint,
-      },
-    },
-  } = useTheming();
+export const isFn = (fn: unknown): fn is Function => typeof fn === 'function';
 
-  const breakpoint = useBreakpoint({
-    breakpoints,
-    breakpointUnit,
-    defaultBreakpoint: defaultBreakpoint as Breakpoint,
-  });
+export const isNullOrEmptyString = (value: unknown) =>
+  value == null || !strHasLength(value);
 
-  return React.useMemo(
-    () =>
-      prefixer(
-        convertStylePropsToStyleObj({ props, style, breakpoint, breakpoints })
-      ),
-    [props, style, breakpoints, breakpoint]
-  );
-};
-
-const filterOutNullOrEmptyStringValues = (value) =>
+export const filterOutNullOrEmptyStringValues = (value) =>
   value != null && (typeof value !== 'string' || strHasLength(value));
```

Comment:
I personally think we can rid of this function (as we can use `isNullOrEmptyString` as replacement). But, if you don't agree, we can simplify it a lot, just returning the opposite of `isNullOrEmptyString`:
 
```suggestion
export const filterOutNullOrEmptyStringValues = (value) => !isNullOrEmptyString(value);
```

Reply:
Good idea, removed.